src/test/util/setup_common.cpp

```diff
@@ -192,7 +196,9 @@ TestingSetup::TestingSetup(const std::string& chainName, const std::vector<const
         throw std::runtime_error(strprintf("ActivateBestChain failed. (%s)", state.ToString()));
     }
 
-    m_node.addrman = std::make_unique<AddrMan>(/* asmap */ std::vector<bool>(), /* deterministic */ false, /* consistency_check_ratio */ 0);
+    m_node.addrman = std::make_unique<AddrMan>(/* asmap */ std::vector<bool>(),
```

Comment:
```suggestion
    m_node.addrman = std::make_unique<AddrMan>(/*asmap=*/ std::vector<bool>(),
```

nit: Could use the clang-tidy understandable syntax when touching this?

Reply:
Done.